app/models/company.rb

```diff
@@ -2,4 +2,17 @@ class Company < ActiveRecord::Base
   validates :name, :city, presence: true
   validates :name, uniqueness: true
   has_many :jobs
+  has_many :contacts
+
+  def show_top_three
```

Comment:
if this method doesn't have anything to do with a specific instance of `Company`, then we should probably make it a class method